src/main/scala/it/pagopa/interop/backendforfrontend/common/system/ApplicationConfiguration.scala

```diff
@@ -9,6 +9,7 @@ import scala.concurrent.duration.FiniteDuration
 object ApplicationConfiguration {
   val config: Config = ConfigFactory.load()
 
+  val pagoPaTenantId: String   = config.getString("backend-for-frontend.pagopa-tenant-id")
```

Comment:
minor suggestion, but this way we can be sure that the value is correct at deploy time instead of waiting the first request
```suggestion
  val pagoPaTenantId: UUID   = UUID.fromString(config.getString("backend-for-frontend.pagopa-tenant-id"))
```
Note: this does not safely handle errors, but neither is `config.getString`, so IMO it is fine